tests/scripts/github-action-helper.sh

```diff
@@ -562,7 +562,9 @@ function get_secret_key() {
 }
 
 function s3cmd() {
-  command timeout 150 s3cmd -v --config=s3cfg --access_key="${S3CMD_ACCESS_KEY}" --secret_key="${S3CMD_SECRET_KEY}" "$@"
+ command timeout 200 bash <<EOF
+  until timeout 10 command s3cmd -v --config=s3cfg --access_key=${S3CMD_ACCESS_KEY} --secret_key=${S3CMD_SECRET_KEY} "$@"; do echo retrying s3cmd >>/dev/stderr ; done
+EOF
 }
```

Comment:
Let's try renaming the function to `run_s3cmd` and changing invocations of it to `run_s3cmd`. Then, the `command` keyword shouldn't need used here and shouldn't be causing us grief. I think that is what's happening